chore: 🤖 adds deploy script (#24)
Psychedelic/DIP721: docs/nft-example.md

```sh
yarn cap:init

> npm run cap:init

Reviewer: That's alright, but the documentation already mentions that'll be using "yarn" as an example and the user can use npm or any other package manager if much prefer. So, to avoid having to show every single command for any other package manager, we accept that yarn is the base of the example, so no need to show the npm example.

"💡 During the guide we'll be using `yarn`, but this can be easily replaced by `npm`, if that's your preference"

```suggestion

```

> 
> npm run cap:init

Reviewer 2: The documentation already mentions that'll be using "yarn" as an example and the user can use npm or any other package manager if much prefer. So, to avoid having to show every single command for any other package manager, we accept that yarn is the base of the example, so no need to show the npm example.

```suggestion
```

```sh
yarn healthcheck

> 
> npm run healthcheck

Reviewer 2: ```suggestion
```

```sh
yarn cap:start

> 
> npm run cap:start

Reviewer 2: ```suggestion
```

```

✨ If everything goes well, you should see the output for a generalist flow, where a user mints a DIP-721 token, gets metadata, get balance, transfers, etc.

### 🌈 Deploying the NFT canister

You can manually deploy the NFT canister by running:

```sh
yarn deploy <local|ic> [reinstall]

> 
> npm run deploy <local|ic> [reinstall]

Reviewer 2: ```suggestion
```

```

> 💡 On creation of a new canister, the controllers have to be set to the principalid of the owner and the canister id. this script automatically sets that up, and redeploys or reinstalls if the canister already exists

Reviewer 2: ```suggestion
💡 On the creation of a new canister, the controllers have to be set to the owners principal and the canister id, as required by the ownership and approvals handling - the script automatically sets that up; And can also redeploy or reinstalls, if the canister already exists.
```

💡 Only use the reinstall if you want the state of the canister completely reset

### 👨🏾‍💻 Development

A develop watcher is available, that builds and runs the healthcheck for any NFT source file changes.

```sh
yarn dev:watch

> 
> npm start dev:watch

Reviewer 2: ```suggestion
```
